packages/openapi3-parser/lib/parser/oas/parseComponentsObject.js

```diff
@@ -130,12 +131,13 @@ function parseComponentsObject(context, element) {
    * @returns ParseResult<ObjectElement>
    * @private
    */
-  const parseComponentObjectMember = (parser) => {
+  const parseComponentObjectMember = R.curry((parser, member) => {
     const parseMember = parseComponentMember(context, parser);
+    const parseMemberOrRef = m => parseReference(member.key.toValue(), () => parseMember(m), context, m.value, false, true);
```

Comment:
nit: I'd extract the `member.key.toValue()` to a variable to avoid calling it multiple times. It is also used below in this function.